### PR TITLE
fix lac spawn error

### PR DESCRIPTION
**Pull request by kinghuin**
### PR types
Bug fixes

### PR changes
Models

### Describe
**LAC报多卡错误**：https://github.com/PaddlePaddle/PaddleNLP/issues/61
**问题复现**：在2.0正式版中必现，在2.0rc1中运行正常
**问题原因**：对于一般loss，例如交叉熵损失函数，它在计算loss的时候是不会引入新参数的。但是crf在计算loss的时候会引入新参数：状态转移矩阵。目前的实现是 network (bigru) → paddle.Model → loss → prepare → fit。DataParallel封装network forward的时候，没有检测到有状态转移矩阵参与计算，因此不会同步状态转移矩阵的反向。但在计算loss的时候，又发现要计算状态转移矩阵的反向，因此发生了错误（2.0正式版新增错误检测）。
**问题解决**：将crfloss在network就计算出来。传给paddle.Model的loss改成一个简单的直接返回函数

### CODE REVIEW

File: examples/lexical_analysis/eval.py [hunk 1]
```diff
@@ -67,7 +68,9 @@ def evaluate(args):
     # Define the model network and metric evaluator
     network = BiGruCrf(args.emb_dim, args.hidden_size, test_dataset.vocab_size,
                        test_dataset.num_labels)
-    model = paddle.Model(network)
+    inputs = InputSpec(shape=(-1, ), dtype="int8", name='inputs')
```
Comment (ZeyuChen) [inline examples/lexical_analysis/eval.py]: Eval的时候就要用InputSpec了？

File: examples/lexical_analysis/eval.py [hunk 2]
```diff
@@ -67,7 +68,9 @@ def evaluate(args):
...
+    lengths = InputSpec(shape=(-1, ), dtype="int8", name='lengths')
+    model = paddle.Model(network, inputs=[inputs, lengths])
```
Comment (wawltor) [inline examples/lexical_analysis/eval.py:73]: 这里使用int8 长度和id都不会超过范围吗？

File: examples/lexical_analysis/predict.py [hunk 3]
```diff
@@ -66,7 +67,9 @@ def infer(args):
     # Define the model network
     network = BiGruCrf(args.emb_dim, args.hidden_size, infer_dataset.vocab_size,
                        infer_dataset.num_labels)
-    model = paddle.Model(network)
+    inputs = InputSpec(shape=(-1, ), dtype="int8", name='inputs')
+    lengths = InputSpec(shape=(-1, ), dtype="int8", name='lengths')
+    model = paddle.Model(network, inputs=[inputs, lengths])
```
Comment (wawltor) [inline examples/lexical_analysis/predict.py:72]: 同上

File: examples/lexical_analysis/train.py [hunk 4]
```diff
@@ -83,15 +84,18 @@ def train(args):
     # Define the model netword and its loss
     network = BiGruCrf(args.emb_dim, args.hidden_size, train_dataset.vocab_size,
                        train_dataset.num_labels)
-    model = paddle.Model(network)
+
+    inputs = InputSpec(shape=(-1, ), dtype="int8", name='inputs')
```
Comment (ZeyuChen) [inline examples/lexical_analysis/train.py]: Train Eval这些都要加InputSpec了吗？

**Review by ZeyuChen — COMMENTED** [review]
为什么新增了很多InputSpec

File: examples/lexical_analysis/eval.py (hunk 2 above)
Comment (kinghuin) [reply examples/lexical_analysis/eval.py:73]: 不会，不过为了便于日后维护，修改为了int16

File: examples/lexical_analysis/predict.py (hunk 3 above)
Comment (kinghuin) [reply examples/lexical_analysis/predict.py:72]: 已修改

File: examples/lexical_analysis/eval.py (hunk 1 above)
Comment (kinghuin) [reply examples/lexical_analysis/eval.py]: 是的，hapi的要求

File: examples/lexical_analysis/train.py (hunk 4 above)
Comment (kinghuin) [reply examples/lexical_analysis/train.py]: 是的，hapi的要求

**Review by ZeyuChen — APPROVED** [review]
LGTM
